packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -523,10 +535,29 @@ export class SwapQuoter {
             gasPrice = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
         }
         // get the relevant orders for the makerAsset
-        let prunedOrders = await this._getSignedOrdersAsync(makerAssetData, takerAssetData);
+        let orders = await this._getSignedOrdersAsync(makerAssetData, takerAssetData);
```

Comment:
Why can't we do this call and the RFQT request in the sample `Promise.all()`?

Reply:
We can and we should!  Per the spec we wrote!  Thank you for noticing this!  Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/5c3929180944eb4c706f6401a9ef81ab411fa264

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/aadcf8fed0ec5ecc202645892180934c02845671